web/src/components/FilingsCalendar.tsx

```diff
@@ -75,34 +75,35 @@ export const FilingsCalendar = (props: Props): ReactElement => {
           <span className="text-bold">{date.format("MMM")}</span> <span>{date.format("YYYY")}</span>
         </div>
         <div>
-          {thisMonthFilings
-            .filter((filing) => {
-              return props.operateReferences[filing.identifier];
-            })
-            .map((filing) => {
-              return (
-                <div key={filing.identifier} className="line-height-1 margin-bottom-1" data-testid="filing">
-                  <Tag backgroundColor="warning-extra-light" isHover isRadiusMd isWrappingText>
+          {date.month() >= getCurrentDate().month() &&
```

Comment:
I'm not sure I understand why this change was made? What does it do?

Reply:
It will prevent past events from showing up. I have now created a function, isCalendarMonthLessThanCurrentMonth, and calling !isCalendarMonthLessThanCurrentMonth() here and using isCalendarMonthLessThanCurrentMonth for the conditional <td>.